elliott

```diff
@@ -377,34 +386,34 @@ advisory with the --add option.
     to advisory 123456:
 
 \b
-    $ elliott --group openshift-3.7 advisory:find-bugs --auto --add 123456
+    $ elliott --group openshift-3.7 advisory:find-bugs --mode sweep --add 123456
 
     List bugs that WOULD be added to an advisory and have set the bro_ok flag on them (NOOP):
 
 \b
-    $ elliott --group openshift-3.7 advisory:find-bugs --auto --flag bro_ok
+    $ elliott --group openshift-3.7 advisory:find-bugs --mode sweep --flag bro_ok
 
     Add two bugs to advisory 123456. Note that --group is not required
     because we're not auto searching:
 
 \b
-    $ elliott advisory:find-bugs --id 8675309 --id 7001337 --add 123456
+    $ elliott advisory:find-bugs --mode list --id 8675309 --id 7001337 --add 123456
 
     Automatically find bugs for openshift-4.1 and attach them to the
     rpm advisory defined in ocp-build-data:
 
 \b
-    $ elliott --group=openshift-4.1 --auto --use-default-advisory rpm
+    $ elliott --group=openshift-4.1 --mode sweep --use-default-advisory rpm
 """
```

Comment:
Last thing. At the bottom of the docstring could you include a note like "Deprecation notice: In 0.1.11 the --auto mode was removed. Use --mode sweep now for equivalent functionality"